.github/workflows/coding-standards.yml

```diff
@@ -54,3 +54,9 @@ jobs:
       # https://github.com/doctrine/.github/issues/3
       - name: "Run PHP_CodeSniffer"
         run: "vendor/bin/phpcs -q --no-colors --report=checkstyle | cs2pr"
+
+      - name: "Output PHP_CodeSniffer autofix diff"
+        if: failure()
+        run: |
+          vendor/bin/phpcbf -q --no-colors --report=checkstyle || true
```

Comment:
This will results in comments in "files changed" tab.

Reply:
Why is `cs2pr` called on line 56, but not on this line?

Reply:
PR updated